stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -1059,12 +1054,12 @@ void VhloBytecodeInterface::write(DictionaryV1Attr attr,
 namespace {
 /// Returns the floating semantics for the given type.
 const llvm::fltSemantics &getFloatSemantics(Type type) {
-  if (type.isa<Float8E4M3FNV1Type>()) return APFloat::Float8E4M3FN();
-  if (type.isa<Float8E5M2V1Type>()) return APFloat::Float8E5M2();
-  if (type.isa<BFloat16V1Type>()) return APFloat::BFloat();
-  if (type.isa<Float16V1Type>()) return APFloat::IEEEhalf();
-  if (type.isa<Float32V1Type>()) return APFloat::IEEEsingle();
-  if (type.isa<Float64V1Type>()) return APFloat::IEEEdouble();
+  if (type.isa<FloatBF16V1Type>()) return APFloat::BFloat();
+  if (type.isa<FloatF16V1Type>()) return APFloat::IEEEhalf();
+  if (type.isa<FloatF32V1Type>()) return APFloat::IEEEsingle();
+  if (type.isa<FloatF64V1Type>()) return APFloat::IEEEdouble();
+  if (type.isa<FloatF8E4M3FNV1Type>()) return APFloat::Float8E4M3FN();
+  if (type.isa<FloatF8E5M2V1Type>()) return APFloat::Float8E5M2();
   llvm_unreachable("non-floating point type used");
```

Comment:
Should we make this `report_fatal_error`? Just noticing this.

Reply:
Yes, this has been addressed in https://github.com/openxla/stablehlo/pull/1185.